geom.js

```diff
@@ -2,31 +2,94 @@ class Rectangle {
   constructor(length, width) {
     this.length = length;
     this.width = width;
+    console.log(this.length + " and " +this.width);
+  }
+  isSquare(){
+    if (this.length==this.width)
+    {
+      console.log("Yes it is square");
+      return true;
+    }
+    else{
+      console.log("It is not a square?");
```

Comment:
While the homework does not ask you to do this, it would make sense to return `false` here.